tests/common.py

```diff
@@ -0,0 +1,133 @@
+from collections.abc import Iterable
+from pathlib import Path
+
+import pyarrow as pa
+
+CUR_DIR = Path(__file__).parent
+
+
+def get_substrait_plan(filename: str) -> str:
+    """
+    Get the substrait plan.
+
+    Parameters:
+        filename:
+            The file to retrieve the substrait plan from.
+
+    Returns:
+        String representation of the json formatted substrait plan.
+    """
+    plan_path = CUR_DIR / "integration" / "queries" / "tpch_substrait_plans" / filename
+
+    with open(plan_path, "r") as f:
+        return f.read()
+
+
+def get_sql(filename: str) -> str:
+    """
+    Get the SQL query
+
+    Parameters:
+        filename:
+            The file to retrieve the SQL from.
+
+    Returns:
+        String representation of the SQL query.
+    """
+    plan_path = CUR_DIR / "integration" / "queries" / "tpch_sql" / filename
+
+    with open(plan_path, "r") as f:
+        return f.read()
+
+
+class SubstraitUtils:
+    """
+    Common utility for substrait integration tests.
+    """
+
+    @staticmethod
+    def get_full_path(file_names: Iterable[str]) -> list[str]:
+        """
+        Get full paths for the TPCH parquet data.
+
+        Parameters:
+            file_names:
+                List of TPCH parquet data file names provided by the test case.
+
+        Returns:
+            List of full paths.
+        """
+        data_dir = CUR_DIR / "data" / "tpch_parquet"
+        full_paths_list = [f"{data_dir}/{dataset}" for dataset in file_names]
+
+        return full_paths_list
+
+    @staticmethod
+    def arrow_sort_tb_values(table: pa.Table, sortby: Iterable[str]) -> pa.Table:
```

Comment:
If you move this to `test_acero_tpch.py` then you can get rid of the `import pyarrow` on this file, leaving it more pure-python and consumer-agnostic.